webpack.config.dev.js

```diff
@@ -6,10 +6,13 @@ const BUILD_DIR = path.resolve(__dirname, './build');
 const SRC_DIR = path.resolve(__dirname, './src');
 
 const config = {
-  entry: ['babel-polyfill', `${SRC_DIR}/index.jsx`],
+  entry: {
+    'app': ['babel-polyfill', `${SRC_DIR}/index.jsx`],
```

Comment:
Should we name the app something other than `app` ?

Reply:
This bundle should contain the JS for the entire "application", which is why it's just simply named app, and the file will be generated as `app.bundle.js`.